modules/sync/opentelemetry/opentelemetry/cas/6b7e445687a785a11c990d8dfeaf3e84c82cac59a22fe87ae37fdc16634e9e0b58fcc6335ae0fb335ba7fe6cb0315116b01650ad273eaac6bc1e5fb69755db22

```diff
@@ -0,0 +1,191 @@
+// Copyright 2023, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opentelemetry.proto.profiles.v1experimental;
+
+import "opentelemetry/proto/common/v1/common.proto";
+import "opentelemetry/proto/resource/v1/resource.proto";
+import "opentelemetry/proto/profiles/v1experimental/pprofextended.proto";
+
+option csharp_namespace = "OpenTelemetry.Proto.Profiles.V1Experimental";
+option java_multiple_files = true;
+option java_package = "io.opentelemetry.proto.profiles.v1experimental";
+option java_outer_classname = "ProfilesProto";
+option go_package = "go.opentelemetry.io/proto/otlp/profiles/v1experimental";
+
+//                Relationships Diagram
+//
+// ┌──────────────────┐                LEGEND
+// │   ProfilesData   │
+// └──────────────────┘            ─────▶ embedded
+//   │
+//   │ 1-n                         ─────▷ referenced by index
+//   ▼
+// ┌──────────────────┐
+// │ ResourceProfiles │
+// └──────────────────┘
+//   │
+//   │ 1-n
+//   ▼
+// ┌──────────────────┐
+// │  ScopeProfiles   │
+// └──────────────────┘
+//   │
+//   │ 1-n
+//   ▼
+// ┌──────────────────┐
+// │ ProfileContainer │
+// └──────────────────┘
+//   │
+//   │ 1-1
+//   ▼
+// ┌──────────────────┐
+// │      Profile     │
+// └──────────────────┘
+//   │                                n-1
+//   │ 1-n         ┌───────────────────────────────────────┐
+//   ▼             │                                       ▽
+// ┌──────────────────┐   1-n   ┌──────────────┐      ┌──────────┐
+// │      Sample      │ ──────▷ │   KeyValue   │      │   Link   │
+// └──────────────────┘         └──────────────┘      └──────────┘
+//   │                    1-n       △      △
+//   │ 1-n        ┌─────────────────┘      │ 1-n
+//   ▽            │                        │
+// ┌──────────────────┐   n-1   ┌──────────────┐
+// │     Location     │ ──────▷ │   Mapping    │
+// └──────────────────┘         └──────────────┘
+//   │
+//   │ 1-n
+//   ▼
+// ┌──────────────────┐
+// │       Line       │
+// └──────────────────┘
+//   │
+//   │ 1-1
+//   ▽
+// ┌──────────────────┐
+// │     Function     │
+// └──────────────────┘
+//
```

Comment:
always love a good ascii diagram

Reply:
Uhh nice! I know there are some tools for this like asciiflow, but hadn't seen it on any managed module yet.

Now I wonder how this renders on the BSR 🤔

Reply:
https://buf.build/opentelemetry/opentelemetry/file/main:opentelemetry/proto/profiles/v1experimental/profiles.proto#L29. Funnily enough, because it's just a floating comment, it doesn't actually exist anywhere in the rendered docs :(!!!